e2e-tests/cypress/support/commandsGeneral.ts

```diff
@@ -0,0 +1,17 @@
+Cypress.Commands.add('setResolution', size => {
```

Comment:
Can you define these functions (and those in other similar files) separately? Then it's possible to type them - right now `size` is an implicit `any` instead of a `Cypress.ViewportPreset.`